### ISSUE TITLE
Resizable layout template or select the template boxes and size

### ISSUE THREAD

**Issue by Harrykd33** [FrSkyRC/ETHOS-Feedback-Community#4474]
The layout templates don't cover what we all want to utilize. If possible the ability to resize the layout screen templates would be great or pick the boxes and sizes even if based on the current boxes we have available.

And, Or a layout with one large model image box and 11 smaller boxes would be appreciated, maybe remove the bottom menu bar as those controls are on the left selection wheel. Thanks .

Basically the smallest box is too large because we would like the ability to have more boxes in a single layout vs switching screens.

**Comment by bsongis-frsky** [comment]
You may draw your own layouts, this is an example:
https://github.com/FrSkyRC/ETHOS-Feedback-Community/tree/1.5/lua/examples/layout

**Comment by Harrykd33** [comment]
I tried getting this working and was unable to, I can't compile a Lua script. maybe create a script with 12 widgets for us common folk.

**Comment by jasalca** [comment]
> I tried getting this working and was unable to, I can't compile a Lua script. maybe create a script with 12 widgets for us common folk.

It can be done now. Send me a email and I will try to explain how to do it.
![8ae2628f-9ba6-417e-b2ff-2795083f77a3](https://github.com/user-attachments/assets/db61c772-70eb-4a52-aa97-03135f516f16)
![856af98b-68a0-462f-9b7a-bd1402e988a0](https://github.com/user-attachments/assets/ffc33aab-da27-4512-9494-67367fcd8c79)

**Comment by Harrykd33** [comment]
Good morning,

Your layout is amazing, please help me understand how to get it working and I'll take it from there.

KevinD
<https://aka.ms/AAb9ysg>
________________________________

**Comment by Harrykd33** [comment]
Please email some instructions doverk@ymail.com

**Comment by jasalca** [comment]
During the morning I've been very busy
